ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/self-test-journey/self-test-journey.ts

```diff
@@ -9,7 +10,7 @@ export class SelfTestJourney {
   static readonly initialStep = SelfTestJourneySteps.SameComputer;
```

Comment:
SelfTestJourney needs to extend JourneyBase

Reply:
It isn't actually used as a `JourneyBase` so I think it's fine not to implement it.